fe/fe-core/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -363,6 +363,16 @@ private String paramsToSql() {
         }
         int len = children.size();
         List<String> result = Lists.newArrayList();
+        if (fnName.getFunction().equalsIgnoreCase("years_diff")
+                || fnName.getFunction().equalsIgnoreCase("months_diff")
+                || fnName.getFunction().equalsIgnoreCase("days_diff")
+                || fnName.getFunction().equalsIgnoreCase("hours_diff")
+                || fnName.getFunction().equalsIgnoreCase("minutes_diff")
+                || fnName.getFunction().equalsIgnoreCase("seconds_diff")) {
+            sb.append(children.get(1).toSql()).append(", ");
+            sb.append(children.get(0).toSql()).append(")");
+            return sb.toString();
+        }
```

Comment:
add comment to explain this is use for nereids and should not be removed

Reply:
done

Reply:
I already add regression case for these functions. If someone remove the above code, the regression cases are failed.